core/modules/replica/HttpProcessorConfig.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * LSST Data Management System
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_HTTPPROCESSORCONFIG_H
+#define LSST_QSERV_HTTPPROCESSORCONFIG_H
+
+// System headers
+#include <string>
+
+// This header declarations
+namespace lsst {
+namespace qserv {
+namespace replica {
+
+/**
+ * Class HttpProcessorConfig captures a configuration of the HTTP service
+ * within the Master Replication Controller to facilitate distributing
+ * the configuration parameters between the service's modules.
+ */
+class HttpProcessorConfig {
+public:
+    /// The normal and the default constructor
+    explicit HttpProcessorConfig(unsigned int workerResponseTimeoutSec_=0,
+                                 unsigned int qservSyncTimeoutSec_=0,
+                                 unsigned int workerReconfigTimeoutSec_=0)
```

Comment:
Noted. It's registered as one of the refactoring efforts in my TODO list: https://confluence.lsstcorp.org/display/DM/Completed%2C+on-going+and+proposed+improvements
It will be done later.